app/src/main/java/com/example/tung/moviedb_07/screen/list_item/ItemMovieViewModel.java

```diff
@@ -26,15 +26,15 @@ public String getTitle() {
     }
 
     public String getReleaseDate() {
-        return mMovie.getReleaseDate();
+        return "Release Date: " + mMovie.getReleaseDate();
     }
 
-    public String getVoteCount() {
-        return String.valueOf(mMovie.getVoteCount());
-    }
-
-    public String getVoteAverage() {
-        return String.valueOf(mMovie.getVoteAverage());
+    public String getVote() {
+        return "Vote : "
+                + String.valueOf(mMovie.getVoteAverage())
+                + " ("
+                + String.valueOf(mMovie.getVoteCount())
```

Comment:
WARNING: [UselessStringValueOf] No need to call String.valueOf to append to a string.
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#UselessStringValueOf